build/bin/sage-logger

```diff
@@ -94,8 +92,7 @@ if [ -n "$SAGE_SILENT_BUILD" -a ${use_prefix} = true ]; then
     # Silent build.
     # Similar to https://www.gnu.org/software/automake/manual/html_node/Automake-Silent-Rules.html#Automake-Silent-Rules
     echo "[$logname] installing. Log file: $logfile"
-    ( exec>> $logfile 2>&1 ; $TIME sh -c "$cmd"; status=$?;
-      [ -r $timefile ] && cat $timefile; exit $status )
+    ( exec>> $logfile 2>&1; time_cmd "$cmd"; status=$?; report_time; exit $status )
```

Comment:
```suggestion
    ( exec>> $logfile 2>&1; time_cmd "$cmd" )
```
or even maybe
```suggestion
    time cmd "$cmd" >> $logfile 2>&$1
```

Also, in L104 below, the time is reported again.

Reply:
One goes to the logfile. One goes to the terminal.